integration/loki_micro_services_delete_test.go

```diff
@@ -23,10 +23,9 @@ func TestMicroServicesDeleteRequest(t *testing.T) {
 			"-target=compactor",
 			"-boltdb.shipper.compactor.compaction-interval=1s",
 			"-boltdb.shipper.compactor.retention-delete-delay=1s",
-			"-boltdb.shipper.compactor.deletion-mode=filter-and-delete",
-			// By default a minute is added to the delete request start time. This compensates for that.
+			// By default, a minute is added to the delete request start time. This compensates for that.
 			"-boltdb.shipper.compactor.delete-request-cancel-period=-60s",
-			"-compactor.allow-deletes=true",
+			"-boltdb.shipper.compactor.deletion-mode=filter-and-delete",
```

Comment:
Can we remove the prefix `boltdb.shipper`? Presumably it will need to work with TSDB indices in the future and I think this will confuse users.